src/flux/client.cr

```diff
@@ -8,6 +8,9 @@ require "./query_result"
 
 class Flux::Client
   Log = ::Log.for(self)
+  {% begin %}
+    alias Params = {{HTTP::Params.resolve? ? HTTP : URI}}::Params
```

Comment:
I think this alias should be private.

```suggestion
    private alias Params = {{HTTP::Params.resolve? ? HTTP : URI}}::Params
```

Reply:
TIL private aliases are a thing! 😃 Thanks for the tip!